docs/downstream-integrations/api/token-gating.md

```diff
@@ -4,7 +4,7 @@ title: Token Gating
 sidebar_position: 4
 ---
 
-The `/access-control` endpoint allows client applications to interact with Collab.Land's token gating engine to verify a wallet address or crypto account with a list of rules based on the token ownership.
+[The `/access-control` endpoint](LINK NEEDED) allows client applications to interact with Collab.Land's token gating engine to verify a wallet address or crypto account with a list of rules based on the token ownership.
```

Comment:
link needed here for access control endpoint.

Reply:
For all APIs, the base URL will be always `https://api.collab.land` for production and `https://api-qa.collab.land` for QA. We don't usually use the full url repetitively.

Reply:
@raymondfeng What is QA? I know it's quality assurance but what does that mean for collabland? How does that fit into code process?

> We don't usually use the full url repetitively.
How would you put a link that goes to the right spot wtihout using a full URL?